inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_vload8.cpp

```diff
@@ -43,6 +44,9 @@ bool EltwiseKernel_vload8::Validate(const Params& params, const optional_params&
         ewParams.output.GetLayout() == DataLayout::fs_b_yx_fsv32)
         return false;
 
+    if ((ewParams.output.GetLayout() == DataLayout::b_fs_yx_fsv4) && (ewParams.output.Feature().v % 8 != 0))
+        return false;
+
```

Comment:
I think it would be better to merge this `if` into previous one to be consistent with the handling of input.

Reply:
Fixed